### PR TITLE
fix(ssg): RHICOMPL-1168 provides no remediation for rsyslog_remote_loghost

### PR DESCRIPTION
**Pull request by akofink**
An initial attempt to disable resolutions of rule rsyslog_remote_loghost for all RHEL versions and profiles.

  ## Secure Coding Practices Checklist GitHub Link
  - https://github.com/RedHatInsights/secure-coding-checklist

  ## Secure Coding Checklist
  - [x] Input Validation
  - [x] Output Encoding
  - [x] Authentication and Password Management
  - [x] Session Management
  - [x] Access Control
  - [x] Cryptographic Practices
  - [x] Error Handling and Logging
  - [x] Data Protection
  - [x] Communication Security
  - [x] System Configuration
  - [x] Database Security
  - [x] File Management
  - [x] Memory Management
  - [x] General Coding Practices

CC @aleccohan @jharting 

Testing with Compliance locally (`false` means no remediation available):
```rb
JSON.parse(RemediationsAPI.new(Account.first).send(:remediations_response, Rule.where(ref_id: 'xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost')).body)
=> {"ssg:rhel8|hipaa|xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost"=>false,
 "ssg:rhel7|c2s|xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost"=>false,
 "ssg:rhel8|cui|xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost"=>false,
 "ssg:rhel7|hipaa|xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost"=>false,
 "ssg:rhel7|anssi_nt28_enhanced|xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost"=>false,
 "ssg:rhel6|cs2|xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost"=>false,
 "ssg:rhel7|nist-800-171-cui|xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost"=>false}
```

Signed-off-by: Andrew Kofink <akofink@redhat.com>

### CODE REVIEW

File: src/connectors/ssg/mock/standard/rsyslog_remote_loghost.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    - no_reboot_needed
+    - restrict_strategy
+    - rsyslog_remote_loghost
+  tasks:
+
+    - name: Set rsyslog remote loghost
+      lineinfile:
+        dest: /etc/rsyslog.conf
+        regexp: ^\*\.\*
+        line: '*.* @@{{ rsyslog_remote_loghost_address }}'
```
Comment (akofink) [inline src/connectors/ssg/mock/standard/rsyslog_remote_loghost.yml:29]: This is the actual rule template. It's easy to see what the issue is :see_no_evil:

File: src/generator/__snapshots__/ssg.integration.js.snap [hunk 2]
```diff
@@ -426,3 +426,94 @@ exports[`generates a simple playbook with uppercase profile name 1`] = `
       command: insights-client
       changed_when: false"
 `;
+
+exports[`ignores rsyslog_remote_loghost rules for compliance remediations 1`] = `
```
Comment (akofink) [inline src/generator/__snapshots__/ssg.integration.js.snap:430]: Notice in this snap, the rule is simply skipped if supplied to the remediations API.

File: src/generator/ssg.integration.js [hunk 3]
```diff
@@ -35,6 +35,24 @@ test('generates a simple playbook with multiple compliance remediation', async (
...
+        }, {
+            id: 'ssg:rhel7|standard|xccdf_org.ssgproject.content_rule_rsyslog_remote_loghost',
+            systems: ['68799a02-8be9-11e8-9eb6-529269fb1459']
+        }]
+    };
+
+    const res = await request
+    .post('/v1/playbook')
+    .send(data)
+    .expect(200);
```
Comment (akofink) [inline src/generator/ssg.integration.js:52]: Without the fix, this returns a 500, reproducing the customer issue.

**Review by aleccohan — COMMENTED** [review]
LGTM, merging

**Pull request merged**